src/components/Link/index.tsx

```diff
@@ -0,0 +1,19 @@
+// Use a wrapper over React Router's Link so that native HTML props can be passed through LeafyGreen MenuItem.
```

Comment:
I think it would be helpful to centralize all the link components (i.e. the ones in `components/styles/StyledLink.tsx`)

Reply:
I just wanted to leave a [link](https://github.com/mongodb/leafygreen-ui/blob/main/packages/select/src/MenuButton.tsx#L20-L33) which shows how LeafyGreen overrides the `Button` styling for their `Select` component. It's not ideal, but we would technically be following their practices 😔 Maybe Sophie can weigh in next week.

Reply:
Applying the same CSS as LeafyGreen to our Dropdown component seems like a good idea to me